web_src/less/_repository.less

```diff
@@ -3172,3 +3172,39 @@ td.blob-excerpt {
 .select-project .item .svg {
     margin-right: .5rem;
 }
+
+#current-card-details {
+    visibility: hidden;
+    position: absolute;
+    right: 0;
+    background-color: #fff;
+    white-space: normal;
+    white-space: initial;
+    z-index: 100;
+    margin: .5rem !important;
+    padding: .5rem !important;
+    width: 320px;
+    height: 60vh;
+    overflow-y: scroll;
+    flex: 0 0 auto;
+    overflow: scroll;
+    display: flex;
+    flex-direction: column;
+}
+
+#issue-search {
+    margin: 1rem .8rem 0 .1rem;
+    width: 300px;
+    position: absolute;
+    right: 0;
+    z-index: 1000;
+    padding: .5rem !important;
+    flex: 0 0 auto;
+    display: flex;
+    flex-direction: column;
+}
+
+#issue-results {
+    margin-top: 4rem;
+    padding: .5rem;
+}
```

Comment:
Can you move these to a new file `web_src/less/features/projects.less` and include it on on top of `index.less`? If possible, use classes instead of ids with a common parent like `.project .class`.

Ideally also move the other styles used in the projects tab to that file.